api/settings.md

```diff
@@ -107,6 +107,35 @@ Status: 200 OK
     "force_update": {
       "enabled": false
     }
+  },
+  "provider": {
+    "parku": {
+      "legals": {
+        "show": [{
+          "text": "By proceeding further you hereby acknowledge and agree the %TERMS_LINK% of ParkU.",
+          "url": "https:\/\/parku.de\/terms\/"
+        }]
+      }
+    },
+    "providerkey": {
+      "image": "http://provider/image.png",
+      "thumbnail": "http://provider/thumbnail.png",
+      "header": "Please provide the data necessary for booking the parking spaces.",
```

Comment:
parking space maybe should be singular here if a user only books one parking space at a time